demo.ipynb

```diff
@@ -10,6 +10,94 @@
fb/TJtHwIECBAgQKAyBASNlVHnzTVLQWMiL2hMMDQJECBAgAABAgQIECBAgACBshMQNJZdSYtqQoLGpByCxgRDkwABAgQIECBAgAABAgQIECg7AUFj2ZW0qCYkaEzKIWhMMDQJECBAgAABAgQIECBAgACBshMQNJZdSYtqQoLGpByCxgRDkwABAgQIECBAgAABAgQIECg7AUFj2ZW0qCYkaEzKIWhMMDQJECBAgAABAgQIECBAgACBshMQNJZdSYtqQoLGpByCxgRDkwABAgQIECBAgAABAgQIECg7AUFj2ZW0qCYkaEzKIWhMMDQJECBAgAABAgQIECBAgACBshMQNJZdSYtqQoLGpBytQeOvfvWrOPLII5M9mgQIECBAgAABAgQIECBAgACB0hfIBo1Tp06NlpaWtsn85je/ieOPP77ttwaBTRUQNCZyrUFjskmTAAECBAgQIECAAAECBAgQIFDWAoLGsi5vQScnaEy4Z8yYEVdddVU0Nja2bZ0zZ0489NBDbb81CBAgQIAAAQIECBAgQIAAAQLlJCBoLKdqbt65CBo78b/11lvjrLPO6uQouwkQIECAAAECBAgQIECAAAECpSkgaCzNuhXjqAWNnVRF0NgJkN0ECBAgQIAAAQIECBAgQIBASQsIGku6fEU1eEFjJ+UQNHYCZDcBAgQIECBAgAABAgQIECBQ0gKCxpIuX1ENXtDYSTmy72tcu3ZtJ0fZTYAAAQIECBAgQIAAAQIECBAoTYEBAwZEdXV1aQ7eqItKQNBYVOUwGAIECBAgQIAAAQIECBAgQIAAAQKlKSBoLM26GTUBAgQIECBAgAABAgQIECBAgACBohIQNBZVOQyGAAECBAgQIECAAAECBAgQIECAQGkKCBpLs25GTYAAAQIECBAgQIAAAQIECBAgQKCoBASNRVUOgyFAgAABAgQIECBAgAABAgQIECBQmgKCxtKsm1ETIECAAAECBAgQIECAAAECBAgQKCoBQWNRlcNgCBAgQIAAAQIECBAgQIAAAQIECJSmgKCxNOtm1AQIECBAgAABAgQIECBAgAABAgSKSkDQWFTlMBgCBAgQIECAAAECBAgQIECAAAECpSkgaCzNuhk1AQIECBAgQIAAAQIECBAgQIAAgaISEDQWVTkMhgABAgQIECBAgAABAgQIECBAgEBpCggaS7NuRk2AAAECBAgQIECAAAECBAgQIECgqAQEjUVVDoMhQIAAAQIECBAgQIAAAQIECBAgUJoCgsbSrJtREyBAgAABAgQIECBAgAABAgQIECgqAUFjUZXDYAgQIECAAAECBAgQIECAAAECBAiUpoCgsTTrZtQECBAgQIAAAQIECBAgQIAAAQIEikpA0FhU5TAYAgQIECBAgAABAgQIECBAgAABAqUpIGgszboZNQECBAgQIECAAAECBAgQIECAAIGiEhA0FlU5DIYAAQIECBAgQIAAAQIECBAgQIBAaQoIGkuzbkZNgAABAgQIECBAgAABAgQIECBAoKgEBI1FVQ6DIUCAAAECBAgQIECAAAECBAgQIFCaAoLG0qybURMgQIAAAQIECBAgQIAAAQIECBAoKgFBY1GVw2AIECBAgAABAgQIECBAgAABAgQIlKaAoLE062bUBAgQIECAAAECBAgQIECAAAECBIpKQNBYVOUwGAIECBAgQIAAAQIECBAgQIAAAQKlKSBoLM26GTUBAgQIECBAgAABAgQIECBAgACBohIQNBZVOQyGAAECBAgQIECAAAECBAgQIECAQGkKCBpLs25GTYAAAQIECBAgQIAAAQIECBAgQKCoBASNRVUOgyFAgAABAgQIECBAgAABAgQIECBQmgKCxtKsm1ETIECAAAECBAgQIECAAAECBAgQKCoBQWNRlcNgCBAgQIAAAQIECBAgQIAAAQIECJSmgKCxNOtm1AQIECBAgAABAgQIECBAgAABAgSKSkDQWFTlMBgCBAgQIECAAAECBAgQIECAAAECpSkgaCzNuhk1AQIECBAgQIAAAQIECBAgQIAAgaISEDQWVTkMhgABAgQIECBAgAABAgQIECBAgEBpCggaS7NuRk2AAAECBAgQIECAAAECBAgQIECgqAQEjUVVDoMhQIAAAQIECBAgQIAAAQIECBAgUJoCgsbSrJtREyBAgAABAgQIECBAgAABAgQIECgqAUFjUZXDYAgQIECAAAECBAgQIECAAAECBAiUpoCgsTTrZtQECBAgQIAAAQIECBAgQIAAAQIEikpA0FhU5TAYAgQIECBAgAABAgQIECBAgAABAqUpIGgszboZNQECBAgQIECAAAECBAgQIECAAIGiEhA0FlU5DIYAAQIECBAgQIAAAQIECBAgQIBAaQoIGkuzbkZNgAABAgQIECBAgAABAgQIECBAoKgEBI1FVQ6DIUCAAAECBAgQIECAAAECBAgQIFCaAoLG0qybURMgQIAAAQIECBAgQIAAAQIECBAoKgFBY1GVw2AIECBAgAABAgQIECBAgAABAgQIlKaAoLE062bUBAgQIECAAAECBAgQIECAAAECBIpKQNBYVOUwGAIECBAgQIAAAQIECBAgQIAAAQKlKSBoLM26GTUBAgQIECBAgAABAgQIECBAgACBohIQNBZVOQyGAAECBAgQIECAAAECBAgQIECAQGkKCBpLs25GTYAAAQIECBAgQIAAAQIECBAgQKCoBASNRVUOgyFAgAABAgQIECBAgAABAgQIECBQmgKCxtKsm1ETIECAAAECBAgQIECAAAECBAgQKCoBQWNRlcNgCBAgQIAAAQIECBAgQIAAAQIECJSmgKCxNOtm1AQIECBAgAABAgQIECBAgAABAgSKSkDQWFTlMBgCBAgQIECAAAECBAgQIECAAAECpSkgaCzNuhk1AQIECBAgQIAAAQIECBAgQIAAgaISEDQWVTkMhgABAgQIECBAgAABAgQIECBAgEBpCvx/CJZwifXuHHkAAAAASUVORK5CYII=)"
```

Comment:
Is there some way you can add instructions for generating this figure, in case we want to modify it in the future? If it's a pain, don't worry about it.

Reply:
I made the figure using the code given at https://github.com/rrtucci/texnn/issues/5